pybillomat/invoices.py

```diff
@@ -128,6 +132,24 @@ def _invoice_xml(
             new_tag.text = str(value)
             invoice_tag.append(new_tag)
 
+    # List Fields
+    for field_name in list_fieldnames:
+        value = locals()[field_name]
+        if value is not None:
+            new_tag = ET.Element(field_name)
+            for subfield in value:
```

Comment:
I would suggest to get rid of `locals()` by not masking the local variable name `field_name` with the loop scoped variable with the same name. It's also confusing that then `value` variable is assigned the `field_name`.

Why not something like this?

```suggestion
    for item in list_fieldnames:
        if field_name is not None:
            new_tag = ET.Element(item)
            for subfield in field_name:
            ...
```

Reply:
I can see that this pattern has been used before. But I would not use that as an argument :)